src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = {lottery.getRandomBall(), lottery.getRandomBall(), lottery.getRandomBall()};
```

Comment:
There is no need to create an array to satisfy the task condition. You can just use for loop to make to lottery.getRandomBall() 3 times, but write this statement only once inside a loop